File: src/scattering/extensions/resonator_2d_domain.py
class Resonator2DDomain(object):
    def __init__(self, H, Lx, Ly, S):
        """

        :param H: waveguide height
        :param Lx: resonator width
        :param Ly: resonator height
        :param S: slit width
        """
        self.H = H
        self.Lx = Lx
        self.Ly = Ly
        self.S = S

File: src-julia/Extensions/Resonator2DDomain.jl
type Resonator2DDomain
    H :: Float64
    Lx :: Float64
    Ly :: Float64
    S :: Float64

    function Resonator2DDomain(H, Lx, Ly, S)
        this = new ()
        this.H = H
        this.Lx = Lx
        this.Ly = Ly
        this.S = S
        return this
    end
end